LayoutTests/fast/frames/frame-depth-limit.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Test iframe depth limit</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <style>
+        body, html {
+            height: 100%;
+        }
+        iframe {
+            width: 100%;
+            height: 100%;
+            display: block;
+        }
+    </style>
+</head>
+<body>
+    <p>Test passes if there is no crash and no timeout.</p>
+    <iframe src="resources/self-referential-iframe.html" scrolling=no></iframe>
+    <script>
+        if (window.testRunner) {
+            testRunner.dumpAsText();
+            testRunner.waitUntilDone();
+        }
+
+        let finishTest = () => {
+            testRunner.notifyDone();
```

Comment:
A little strange that we check for `window.testRunner` in the code running at the start of the test, but not at the finish. I suggest having the check here too.

Could use DOM mutation to add a message here so we would not have to deduce from lack of hang that we got done. Especially useful if you run this in browser instead of with `testRunner`.